.changeset/lemon-actors-search.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+build(deps): bump handlebars from 4.7.7 to 4.7.8
```

Comment:
It think that `abi-typegen` and `fuels` packages should be bumped here no? Looks like a bug.

Reply:
Updated in [`23192ee` (#2588)](https://github.com/FuelLabs/fuels-ts/pull/2588/commits/23192ee53b37e8def6b0b0ba666a278a1aa29fca)